Source/Core/DolphinQt/Config/NewPatchDialog.cpp

```diff
@@ -18,6 +18,14 @@
 #include "Core/PatchEngine.h"
 #include "DolphinQt/QtUtils/ModalMessageBox.h"
 
+struct NewPatchEntry
+{
+  QLineEdit* address = nullptr;
+  QLineEdit* value = nullptr;
+  QLineEdit* comparand = nullptr;
+  PatchEngine::PatchEntry entry;
+};
```

Comment:
To avoid someone re-creating the same mistake in the future, please make this struct non-copyable and non-movable, with a comment saying why.